runhouse/rns/defaults.py

```diff
@@ -31,7 +31,7 @@ class Defaults:
         "default_autostop": -1,
         "use_spot": False,
         "use_local_configs": True,
-        "disable_data_collection": False,
+        "enable_telemetry_collection": True,
```

Comment:
should we keep this naming consistent with the server's logic (`disable_telemetry` or something like that)?